lib/ui5Framework/npm/Registry.js

```diff
@@ -49,19 +49,21 @@ class Registry {
 	}
 	async _getPacoteOptions() {
 		if (!this._npmConfig) {
-			const {default: libnpmconfig} = await import("libnpmconfig");
-			const opts = {
-				cache: this._cacheDir,
-			};
-			const config = libnpmconfig.read(opts, {
-				cwd: this._cwd
-			}).toJSON();
+			const {default: Config} = await import("@npmcli/config");
+			const {
+				default: {flatten, definitions, shorthands, defaults},
+			} = await import("@npmcli/config/lib/definitions/index.js");
 
-			// Rename https-proxy to httpsProxy so that it is picked up by npm-registry-fetch (via pacote)
-			if (config["https-proxy"]) {
-				config.httpsProxy = config["https-proxy"];
-				delete config["https-proxy"];
-			}
+			const configuration = new Config({
+				npmPath: this._cwd,
```

Comment:
I tested this and couldn't find a scenario where this makes a difference. It 

However, since we actively set `this._cwd` to the project's root directory in the past, I think we should still do the same here. This also seems to align with the expectations of `@npmcli/config`:  https://github.com/npm/cli/blob/config-v6.2.1/workspaces/config/lib/index.js#L659

Reply:
Done